gemfiles/rails_4_2.gemfile

```diff
@@ -1,36 +1,36 @@
 # This file was generated by Appraisal
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
```

Comment:
In that case I will add `Bundler/OrderedGems` in the rubocop yml to exclude the `gemfiles` or we not interested in that cop altogether?

Reply:
I think that's fine to have that cop. I think we should exclude Rubocop from parsing `*.gemfiles` altogether, though, since they will never be modified on their own.